libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyParserTests.java

```diff
@@ -52,6 +52,35 @@ public void testPolicyBuilderOnExternalPlugin() throws IOException {
         assertEquals(expected, parsedPolicy);
     }
 
+    public void testParseFile() throws IOException {
+        Policy parsedPolicy = new PolicyParser(new ByteArrayInputStream("""
+            entitlement-module-name:
+              - file:
+                  path: test/path/to/file
+                  actions:
+                    - read
```

Comment:
We should put all possible actions in here to ensure they can all be parsed.

Reply:
Same, will remove it